core/models/Auth.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  api: {
+    "post auth": [
+      async (ctx) => {
+        try {
+          let operator = await AuthService.getOperatorByCredentials(ctx.request.body.email, ctx.request.body.password);
+          ctx.cookies.set("jwt", AuthService.generateToken(operator));
+          ctx.response.status=200;
+        } catch (err) {
+          ctx.throw(403, "Wrong email or password");
```

Comment:
It's better to use 401 instead of 403

Reply:
And, it's wiser to use error message `err.message` instead of putting new string here